fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerTest.java

```diff
@@ -326,16 +326,23 @@ private void singleRebuild(@Nonnull List<TestRecords1Proto.MySimpleRecord> recor
                 .setMetaData(metaData)
                 .setIndex(index)
                 .setSubspace(subspace)
-                .setLimit(20)
-                .setMaxRetries(Integer.MAX_VALUE)
-                .setRecordsPerSecond(OnlineIndexer.DEFAULT_RECORDS_PER_SECOND * 100)
-                .setTimer(timer);
-        if (ThreadLocalRandom.current().nextBoolean()) {
-            // randomly enable the progress logging to ensure that it doesn't throw exceptions,
-            // or otherwise disrupt the build.
-            LOGGER.info("Setting progress log interval");
-            builder.setProgressLogIntervalMillis(0);
-        }
+                .setConfigLoader(old -> {
+                    if (old == null) {
```

Comment:
Wait, when is `old` `null`? Isn't the config set to, essentially, the default config if a loader is specified?

Reply:
Good catch, I used to not initialize it, until I decided to do so. This predates that. fixed now.